README.md

```diff
@@ -23,13 +23,12 @@ Using Gradle makes building WPILib very straightforward. It only has a few depen
 
 ## Requirements
 
-- A C++ compiler
-    - On Linux, GCC works fine
-    - On Windows, you need Visual Studio 2019 (the free community edition works fine).
-      Make sure to select the C++ Programming Language for installation
-- [ARM Compiler Toolchain](https://github.com/wpilibsuite/roborio-toolchain/releases)
-  * Note that for 2020 and beyond, you should use version 7 or greater of GCC
-- Doxygen (Only required if you want to build the C++ documentation)
+- C++ compiler
+    - On Linux, install GCC
```

Comment:
Maybe add a bullet point for Mac users asking them to install the Xcode command-line build tools by running `xcode-select --install`?

Reply:
Yep.